.editorconfig

```diff
@@ -6,7 +6,7 @@ indent_style = space
 insert_final_newline = false
 max_line_length = 120
 tab_width = 4
-ij_continuation_indent_size = 8
```

Comment:
I think it doesn't help, but I'm rited of reformatting code the way it does (8 spaces :sdfwe@!!@!@@!)